articles/libraries/auth0js/index.md

```diff
@@ -1,28 +1,28 @@
 ---
 section: libraries
 toc: true
-description: How to install, initialize and use auth0.js
+description: How to install, initialize and use auth0.js v8
 url: /libraries/auth0js
 ---
 
-# Auth0.js Reference
+# Auth0.js v8 Reference
 
 Auth0.js is a client-side library for [Auth0](http://auth0.com), for use in your web apps. It allows you to trigger the authentication process and parse the [JSON Web Token](http://openid.net/specs/draft-jones-json-web-token-07.html) (JWT) with just the Auth0 `clientID`. Once you have the JWT, you can use it to authenticate requests to your HTTP API and validate the JWT in your server-side logic with the `clientSecret`.
```

Comment:
Also `validate the JWT in your server-side logic with the clientSecret` is not only not accurate but wrong for the new pipeline since is not the secret of the client but the api signing secret